app/services/ui/host_description.rb

```diff
@@ -16,4 +16,20 @@ def self.reduce_providers(customization_point)
       end
     end
   end
+
```

Comment:
Can we move the hostgroup description into a separate file? I would vote for `app/services/ui/hostgroup_description.rb`